docs/mcp/client.md

```diff
@@ -136,7 +136,7 @@ MCP also offers [stdio transport](https://spec.modelcontextprotocol.io/specifica
 
 In this example [mcp-run-python](https://github.com/pydantic/mcp-run-python) is used as the MCP server.
 
-```python {title="mcp_stdio_client.py"}
+```python {title="mcp_stdio_client.py" test="skip"}
```

Comment:
let's not include this in this PR, and in the PR where we do this, we should also drop the deno requirement

Reply:
I pushed the fix for the mcp error here https://github.com/pydantic/pydantic-ai/pull/3779